Scripts/JSONFileToCSV/JSONFileToCSV.py

```diff
@@ -0,0 +1,54 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import json
+import io
+import csv
+
+
+def json_to_csv(data, delimiter):
+    si = io.BytesIO()
+    cw = csv.writer(si, delimiter=delimiter)
+    keys = list(data[0].keys())
+    cw.writerow(keys)
+    for d in data:
+        val_lst = []
+        for k in keys:
+            val_lst.append(d[k])
+        cw.writerow(val_lst)
+    return si.getvalue().strip("\r\n")
+
+
+def main(entry_id, out_filename, delimiter):
+    if isinstance(entry_id, list):
```

Comment:
need to call `argToList` on entry_id, no?

Reply:
actually this can be removed.
since the `is_array` flag is not set, entry_id will never be a list